### DISCUSSION TITLE
Old Broken Website Keeps Showing Up

### DISCUSSION THREAD

**Discussion by QuintinN** [community/community#77262] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I have a repository for my personal website. Every change i make, it shows the same broken old web page and I don't know why. I need help from the internal service team https://github.com/QuintinN/quintinn.github.io. 
Things I have tried: 
- Deleting the repository and any repository associated
- Clearing my internet browser cache 
- Built on gh-pages branch and master branch 
- I'm considering just deleting my account and creating a  new account to host the webpage

**Comment by peterroth** [comment]
Did you follow the below guide to set the branch you want to build your site from?
https://docs.github.com/en/pages/getting-started-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site#publishing-from-a-branch
